@here/harp-mapview/lib/Tile.ts

```diff
@@ -32,6 +32,13 @@ export type TileObject = THREE.Object3D & {
      * Distance of this object from the [[Tile]]'s center.
      */
     displacement?: THREE.Vector3;
+
+    /**
+     * This stores the THREE.Object3D renderOrder property, we need to back it up because we need to
+     * reduce it if the tile is used as fall back. When it is used normally, the renderOrder needs
+     * to be reset.
+     */
+    backupRenderOrder?: number;
```

Comment:
If this is an internal property that users are not supposed to use and modify you should mark it with @hidden in comment and rename it to _backupRenderOrder

Reply:
resolved in latest commit: b9c2c3920d9b0d435e8ade3cb8b1ff7b722bfb88